utils/resize.sh

```diff
@@ -29,14 +29,11 @@ if [ $(readlink -f /dev/xvda) = "/dev/xvda" ]
 then
   # Rewrite the partition table so that the partition takes up all the space that it can.
   sudo growpart /dev/xvda 1
- 
-  # Expand the size of the file system.
-  sudo resize2fs /dev/xvda1
-
 else
   # Rewrite the partition table so that the partition takes up all the space that it can.
   sudo growpart /dev/nvme0n1 1
+fi
+
+# Expand the size of the file system
+sudo xfs_growfs /
```

Comment:
Need to use this instead of resize2fs now that root is on XFS in the newer images